src/displayapp/screens/ApplicationList.h

```diff
@@ -38,8 +38,7 @@ namespace Pinetime {
 
         static constexpr int appsPerScreen = 6;
 
-        // Increment this when more space is needed
-        static constexpr int nScreens = (UserAppTypes::Count / appsPerScreen) + 1;
+        static constexpr int nScreens = CEIL_DIV(UserAppTypes::Count, appsPerScreen);
```

Comment:
I'm not sure where this macro comes from. I would prefer using `(UserAppTypes::Count - 1) / appsPerScreen + 1`. You should also remove the comment above as it no longer makes sense.

Reply:
Beaten you to it with the comment, it's already gone :)
The macro is from the NRF SDK. Personally I think it's clearer semantically but I'll change it to what you suggest

Reply:
Thank you! While I do think it is clearer what it's doing with the macro, I don't think that it makes sense to use the nRF SDK here.

Reply:
That's a valid point, probably not a great dependency for this file